s3-get-files.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Log file path
+log_file="/root/SCRIPTS/logfile.txt"
+
+# Function to log messages with timestamp
+log() {
+    echo "$(date +"%Y-%m-%d %H:%M:%S") $1" >> "$log_file"
+}
+
+# Get the files in the directory
+log "Listing files in the S3 directory"
+files=$(s3cmd ls s3://infinity-91872 | cut -d/ -f4)
```

Comment:
This line cannot retrieve the file names correctly if they have a space in their names.